bukkit-helper-119/src/main/java/org/dynmap/bukkit/helper/v119/BukkitVersionHelperSpigot119.java

```diff
@@ -199,245 +194,242 @@ public MapChunkCache getChunkCache(BukkitWorld dw, List<DynmapChunk> chunks) {
-        return tileent.getBlockPos().getZ();
-    }
-
-    @Override
-    public Object readTileEntityNBT(Object te) {
-        BlockEntity tileent = (BlockEntity) te;
-        return tileent.saveWithId();
-    }
-
-    @Override
-    public Object getFieldValue(Object nbt, String field) {
-        CompoundTag rec = (CompoundTag) nbt;
-        Tag val = rec.get(field);
-        if (val == null) return null;
-        if (val instanceof ByteTag) {
-            return ((ByteTag) val).getAsByte();
-        } else if (val instanceof ShortTag) {
-            return ((ShortTag) val).getAsShort();
-        } else if (val instanceof IntTag) {
-            return ((IntTag) val).getAsInt();
-        } else if (val instanceof LongTag) {
-            return ((LongTag) val).getAsLong();
-        } else if (val instanceof FloatTag) {
-            return ((FloatTag) val).getAsFloat();
-        } else if (val instanceof DoubleTag) {
-            return ((DoubleTag) val).getAsDouble();
-        } else if (val instanceof ByteArrayTag) {
-            return ((ByteArrayTag) val).getAsByteArray();
-        } else if (val instanceof StringTag) {
-            return val.getAsString();
-        } else if (val instanceof IntArrayTag) {
-            return ((IntArrayTag) val).getAsIntArray();
+        return getBiomeReg().getKey((Biome)bb).toString();
+	}
+
+	@Override
+	public Object getUnloadQueue(World world) {
+		Log.warning("getUnloadQueue not implemented yet");
+		// TODO Auto-generated method stub
+		return null;
+	}
+
+	@Override
+	public boolean isInUnloadQueue(Object unloadqueue, int x, int z) {
+		Log.warning("isInUnloadQueue not implemented yet");
+		// TODO Auto-generated method stub
+		return false;
+	}
+
+	@Override
+	public Object[] getBiomeBaseFromSnapshot(ChunkSnapshot css) {
+		Log.warning("getBiomeBaseFromSnapshot not implemented yet");
+		// TODO Auto-generated method stub
+		return new Object[256];
+	}
+
+	@Override
+	public long getInhabitedTicks(Chunk c) {
+		return ((CraftChunk) c).getHandle().getInhabitedTime();
+	}
+
+	@Override
+	public Map<?, ?> getTileEntitiesForChunk(Chunk c) {
+		return ((CraftChunk) c).getHandle().blockEntities;
+	}
+
+	@Override
+	public int getTileEntityX(Object te) {
+		BlockEntity tileent = (BlockEntity) te;
+		return tileent.getBlockPos().getX();
+	}
+
+	@Override
+	public int getTileEntityY(Object te) {
+		BlockEntity tileent = (BlockEntity) te;
+		return tileent.getBlockPos().getY();
+	}
+
+	@Override
+	public int getTileEntityZ(Object te) {
+		BlockEntity tileent = (BlockEntity) te;
+		return tileent.getBlockPos().getZ();
+	}
+
+	@Override
+	public Object readTileEntityNBT(Object te) {
+		BlockEntity tileent = (BlockEntity) te;
+		return tileent.saveWithId();
+	}
+
+	@Override
+	public Object getFieldValue(Object nbt, String field) {
+		CompoundTag rec = (CompoundTag) nbt;
+		Tag val = rec.get(field);
+        if(val == null) return null;
+        if(val instanceof ByteTag) {
```

Comment:
The space usage gets inconsistent as of these statements cuz of `if (` -> `if(`, and other unnecessary extra spaces were added before these lines.

Reply:
I do 0 changes in spaces here, if mike will want these lines to be pretty, he can do that himself, maybe you meant other line but in method getFieldValue there are 0 changes in space usage

Reply:
https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-personal-account-settings/managing-your-tab-size-rendering-preference
try this and set default tab size to 4 spaces, then it'll be the same if you really want